src/networkpolicy/networkPolicy.go

```diff
@@ -2147,8 +2147,10 @@ func StartNetworkLogRcvr() {
 	for {
 		if cfg.GetCfgNetworkLogFrom() == "hubble" {
 			plugin.StartHubbleRelay(NetworkStopChan /* &NetworkWaitG, */, cfg.GetCfgCiliumHubble())
-		} else if cfg.GetCfgNetworkLogFrom() == "kafka" {
-			feedconsumer.StartConsumer()
+		} else if cfg.GetCfgNetworkLogFrom() == "feed-consumer" {
+			fc.ConsumerMutex.Lock()
+			fc.StartConsumer()
+			fc.ConsumerMutex.Unlock()
```

Comment:
This lock here is necessary since the `fc.StartConsumer()` updates few global variables.